.github/deployment/front/affine.nginx.conf

```diff
@@ -1,12 +1,14 @@
 server {
   listen 8080;
-  location / {
-    root /app/dist;
-    try_files $uri $uri/ /index.html;
+  location /admin {
+    root /app/;
```

Comment:
/app/admin?

Reply:
The `$uri` in `try_files` is /admin. Setting the root to `/app/` makes `try_files $uri` become `/app/admin/xxx.js`.